llvm/test/Transforms/LoopVectorize/pr66616.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: opt -passes="require<scalar-evolution>,print<scalar-evolution>,loop-vectorize" --verify-scev --verify-scev-strict -S < %s | FileCheck %s
```

Comment:
Could this be moved to `scev-invalidation.ll` to keep similar tests together?

Reply:
The `require<scalar-evolution>` here is redundant.

Also, do not add `-verify-scev-strict` to tests. Failures reported by this option are usually spurious.